qiskit/providers/aer/noise/errors/standard_errors.py

```diff
@@ -321,11 +265,17 @@ def depolarizing_error(param, num_qubits, standard_gates=True):
     probs = [prob_iden] + (num_terms - 1) * [prob_pauli]
     # Generate pauli strings. The order doesn't matter as long
     # as the all identity string is first.
-    paulis = [
-        "".join(tup)
-        for tup in it.product(['I', 'X', 'Y', 'Z'], repeat=num_qubits)
-    ]
-    return pauli_error(zip(paulis, probs), standard_gates=standard_gates)
+
+    circs = []
+    for pauli_list in it.product([IGate(), XGate(), YGate(), ZGate()], repeat=num_qubits):
+        qc = QuantumCircuit(num_qubits)
+        for q, pauli in enumerate(pauli_list):
+            if not standard_gates:
+                pauli = UnitaryGate(pauli.to_matrix())
```

Comment:
This change should be removed and replace with above code snippet. We don't want to convert depolarzing error to unitary gates, sicne it will no longer run on the stabilizer simulator.